File: matrices-simple/javascript/matriz.js
(() => {

    let people = [];
    let numPeople = parseInt(prompt("Cuantas personas desea registrar?: "));

    for (let i = 0; i < numPeople; i++) {

        alert(`Persona ${i + 1}: `);
        let name = String(prompt("Nombre: "));
        let age = parseInt(prompt("Edad: "));
        let note = parseFloat(prompt("Nota: "));

        people.push([name, age, note]);
    }

    console.log('Registro de personas')
    for (let i = 0; i < people.length; i++) {
        console.log(`Nombre: ${people[i][0]}, Edad: ${people[i][1]}, Nota: ${people[i][2]}`)
    }
    console.log('------')
    console.log('Orden por nota')
    people.sort((a, b) => b[2] - a[2]);
    for (let i = 0; i < people.length; i++) {
        console.log(`Nombre: ${people[i][0]}, Edad: ${people[i][1]}, Nota: ${people[i][2]}`);
    }


})();

File: matrices-simple/python/matriz.py
people = []

num_people = int(input("¿Cuántas personas desea registrar?: "))

for i in range(num_people):
    print(f"\nPersona {i + 1}:")
    name = input("Nombre: ")
    age = int(input("Edad: "))
    note = float(input("Nota: "))

    people.append([name, age, note])

print("\nRegistro de personas:")
for person in people:
    print(f"Nombre: {person[0]}, Edad: {person[1]}, Nota: {person[2]}")

people.sort(key=lambda x: x[2], reverse=True)

print("\n------")
print("Orden por nota:")
for person in people:
    print(f"Nombre: {person[0]}, Edad: {person[1]}, Nota: {person[2]}")
